index.html

```diff
@@ -0,0 +1,139 @@
+            <div class="menuline line-2"></div>
+            <div class="menuline line-3"></div>
+          </div>
+          <ul class="menuitems">
+            <li class="menuitem menuitem-1">
+                <a href="./index.html">Home</a>
+            </li>
+            <li class="menuitem menuitem-2">
+              <a href="./about.html">About</a>
+            </li>
+            <li class="menuitem menuitem-3">
+              <a href="#">lorem</a>
+            </li>
+            <li class="menuitem menuitem-4">
+              <a href="#">Ipsum</a>
+            </li>
+          </ul>
+          <div class="navmenubottom"></div>
+        </nav>
+        <nav class="header-desktop">
+          <div class="headerimgcontainer"><a href="./index.html"><img class="headerimg" src="./imgs/E3logowhite.png" alt=""></a></div>
+          <ul class="headercontainer">
+            <li class="header-desktop1"><a href="./index.html">Home</a></li>
+            <li class="header-desktop2"><a href="./about.html">About</a></li>
+            <li class="header-desktop3"><a href="#">Lorem</a></li>
+            <li class="header-desktop4"><a href="#">Ipsum</a></li>
+          </ul>
+          <div class="hbuttoncontainer"><button class="headerbutton">News</button></div>
+        </nav>
+      </header>
+      <main>
+        <div class="headline">
+          <p class="ph1">"Stay in the game!"</p>
+          <h1 class="p1">
+              Worldwide broadcast E3 2021
+          </h1>
+          <h2  class="pd1">
+            Worldwide broadcast E3 2021
+          </h2>
+          <p class="pheadline">
+            Hardware manufacturers, software developers, 
+            and publishers from the video game industry would have 
+            presented new and upcoming products. 
+          </p>
+          <h2 class="date">June.9.2020 (TUE) ~ 11 (FRY)</h2>
+          <p class="location">@ Los Angeles Convention Center, Los Angeles California</p>
+        </div>
+        <section class="events">
+          <h2 class="mainevents">Main Events</h2>
+          <h2 class="eventsline1">____</h2>
+          <div class="eventcard eventcard1">
+            <img class="eventimg" src="./imgs/SONY-SQUARE1.png" alt="">
+            <h2 class="event-tittle">Sony conference</h2>
+            <p class="eventdescpt">Sony ran its major reveal of the PlayStation 5 console with all 
+              the new games and sooner releases</p>
+          </div>
+          <div class="eventcard eventcard2">
+            <img class="eventimg" src="./imgs/nintendologo1.png" alt="">
+            <h2 class="event-tittle">Nintendo conference</h2>
+            <p class="eventdescpt">Nintendo surprises with Breath of the Wild 2, Banjo-Kazooie and more
+              This year's Nintendo Direct was packed with reveals</p>
+          </div>
+          <div class="eventcard eventcard3">
+            <img class="eventimg" src="./imgs/Electronic_Arts-Logo.png" alt="">
```

Comment:
- [ ] A good practice for `<img>` tag is always add a value for `alt` tag, please add it. 🐻